src/components/SelectionList/SearchTableHeader.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import {View} from 'react-native';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useStyleUtils from '@hooks/useStyleUtils';
+import useThemeStyles from '@hooks/useThemeStyles';
+import useWindowDimensions from '@hooks/useWindowDimensions';
+import CONST from '@src/CONST';
+
+type SearchTableHeaderProps = {
+    /** Whether we should show the merchant or description column */
+    shouldShowMerchant: boolean;
+};
+
+function SearchTableHeader({shouldShowMerchant}: SearchTableHeaderProps) {
+    const styles = useThemeStyles();
+    const StyleUtils = useStyleUtils();
+    const {isSmallScreenWidth, isMediumScreenWidth} = useWindowDimensions();
+    const {translate} = useLocalize();
+    const displayNarrowVersion = isMediumScreenWidth || isSmallScreenWidth;
+
+    if (displayNarrowVersion) {
+        return;
+    }
+
+    return (
+        <View style={[styles.ph5, styles.pb3]}>
+            <View style={[styles.flex1, styles.flexRow, styles.gap3, styles.ph4]}>
+                <View style={[StyleUtils.getSearchTableColumnStyles(CONST.SEARCH_TABLE_COLUMNS.DATE)]}>
+                    <Text style={[styles.mutedNormalTextLabel]}>{translate('common.date')}</Text>
+                </View>
+                <View style={[StyleUtils.getSearchTableColumnStyles(CONST.SEARCH_TABLE_COLUMNS.MERCHANT)]}>
+                    <Text style={[styles.mutedNormalTextLabel]}>{translate(shouldShowMerchant ? 'common.merchant' : 'common.description')}</Text>
+                </View>
```

Comment:
Could you explain why merchant/description are grouped

Reply:
Right now we're showing one or the other, following this logic in the HL

>The Merchant and Description columns will show based on the same logic already in use for the expense previews in Inbox. In other words, Merchant will always be shown for workspace expenses, because Merchant is required and triggers violations. Off-workspace, we’ll show Merchant as long as at least one value isn’t “Request” or “Unknown merchant”, based on existing logic. In all other cases, we’ll show Description as long as at least one expense has one (and if a description is long, we’ll truncate with an ellipsis, just like any other column).